tools/generate_tables_sql_and_testNames.php

```diff
@@ -1,118 +1,239 @@
 #!/usr/bin/php
 <?php
-
 /**
- * generate_tables_sql.php takes the ip_output.txt file generated from
- * quickform_parser.php and outputs an sql build file for the table of each
- * instrument it finds in the ip_output.txt file.  These sql files are output
- * to the tables_sql/ subdirectory.
+ * Behavioural
+ *
+ * PHP Version 7
+ *
+ * The generate_tables_sql_and_testNames.php takes the {instrument_name}.linst file
+ * from /instrument_manager and installs an instrument for the table of each
+ * instrument it finds in the {instrument_name}.linst file.
+ * These sql files are output to the tables_sql/ subdirectory.
+ *
+ * Ex cmd with SQL execution:
+ * php generate_tables_sql_and_testNames.php < ../instruments/[instrument_name].linst
  *
- * ex cmd:  php generate_tables_sql.php
+ * Ex cmd without SQL execution:
+ * php generate_tables_sql_and_testNames.php <
+ * ../instruments/[instrument_name].linst false
  *
- * @package behavioural
+ * @category Tools
+ * @package  Behavioural
+ * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
  */
 
-//Ensure php version compatability
-//taken from php.net notes
-
 // require all relevant OO class libraries
 require_once __DIR__ . "/../vendor/autoload.php";
 require_once "../php/libraries/Database.class.inc";
 require_once "../php/libraries/NDB_Config.class.inc";
 require_once "../php/libraries/NDB_BVL_Instrument.class.inc";
 
-$data = stream_get_contents(STDIN);
+$execute_mysql_query = ($argv[1] ?? true) === true;
 
-$instruments=explode("{-@-}",trim($data));
+$data          = stream_get_contents($argv[0]);
+$instruments   = explode("{-@-}", trim($data));
```

Comment:
I don't fully understand what's happening here. Are we getting a linst file (delimited by {@}) with multiple instruments in it (each delimited by {-@-}) ?